README.md

```diff
@@ -1,15 +1,15 @@
 # sonarcloud-circleci-orb
 Support of SonarScanner CLI in CircleCI
 
-## SonarCloud Orb
-The SonarCloud Orb can be used with any linux based docker image that includes the command line tools `curl` and `unzip`.
+## SonarQube Cloud Orb
+The SonarQube Cloud Orb can be used with any linux based docker image that includes the command line tools `curl` and `unzip`.
 
-To connect to your SonarCloud project on `sonarcloud.io` you need to setup an api token. We recommend to setup a CircleCI context in your organization named `sonarcloud` that contains a variable with key `SONAR_TOKEN` and the api token as the value.
+To connect to your SonarQube Cloud project on `sonarcloud.io` you need to setup an api token. We recommend to setup a CircleCI context in your organization named `sonarcloud` that contains a variable with key `SONAR_TOKEN` and the api token as the value.
 ### Usage examples
 ```yaml
 version: 2.1
 orbs:
-  sonarcloud: sonarsource/sonarcloud@1.0.3
+  sonarcloud: sonarsource/sonarcloud@2.0.0
```

Comment:
I bumped to 2.0.0 to align with the orb page
https://circleci.com/developer/orbs/orb/sonarsource/sonarcloud